opencti-platform/opencti-front/src/private/components/cases/Resolver.js

```diff
@@ -35,11 +34,11 @@ const Resolver = () => {
             if (props.case) {
               let redirectLink;
               const { case: caseEntity } = props;
-              if (caseEntity.case_type === 'feedback') {
+              if (caseEntity.entity_type === 'feedback') {
                 redirectLink = `/dashboard/cases/feedbacks/${caseEntity.id}`;
-              } else if (caseEntity.case_type === 'incident') {
+              } else if (caseEntity.entity_type === 'incident') {
                 redirectLink = `/dashboard/cases/incidents/${caseEntity.id}`;
-              } else if (caseEntity.case_type === 'rfi') {
+              } else if (caseEntity.entity_type === 'rfi') {
```

Comment:
how does this `ResolverCaseQuery` work ? does `entity_type` have the same value as `case_type` ?